UM/ColorGenerator.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) 2015 Ultimaker B.V.
 # Uranium is released under the terms of the AGPLv3 or higher.
 
+#pylint: disable=bad-whitespace
```

Comment:
Unfortunately, pylint only ignores this style of whitespace for dicts, not lists. So it is still needed.